res/conf

```diff
@@ -34,6 +34,9 @@ XINITRC_LAUNCH=false
 # Prints available WM/DE each on new line instead of printing on single line.
 VERTICAL_SELECTION=false
 
+# Horizontal offset if VERTICAL_SELECTION is in use
+#INDENT_SELECTION=0
+
```

Comment:
Please, remove it from default `res/conf`, it is default configuration. Some distros will overwrite users' configurations, if this file is changed (yeah, I know...).